clj-refactor.el

```diff
@@ -8,7 +8,7 @@
 ;;         Benedek Fazekas <benedek.fazekas@gmail.com>
 ;; Version: 2.5.0-snapshot
 ;; Keywords: convenience, clojure, cider
-;; Package-Requires: ((emacs "25.1") (seq "2.19") (yasnippet "0.6.1") (paredit "24") (multiple-cursors "1.2.2") (clojure-mode "5.6.1") (cider "0.17.0") (edn "1.1.2") (inflections "2.3") (hydra "0.13.2"))
+;; Package-Requires: ((emacs "25.1") (seq "2.19") (yasnippet "0.6.1") (paredit "24") (multiple-cursors "1.2.2") (clojure-mode "5.6.1") (cider "0.24.0-snapshot") (edn "1.1.2") (inflections "2.3") (hydra "0.13.2"))
```

Comment:
I'm not sure what cider version should we depend on here?

Reply:
Generally it should the latest stable CIDER release.

Reply:
I agree with @bbatsov that we should change this to be the latest stable release, and aim to update it more frequently than we've done in the past.

If we depend on the snapshot we're giving ourselves very little lead-time to react to the occasional breaking change in CIDER.